Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -82,9 +87,10 @@
   private final DartServerRootsHandler myRootsHandler = new DartServerRootsHandler();
   private final Map<String, Long> myFilePathWithOverlaidContentToTimestamp = new THashMap<String, Long>();
   private final List<String> myPriorityFiles = new ArrayList<String>();
+  private final Set<String> myFilePathsWithUnsentChanges = Sets.newHashSet();
 
   @NotNull private final Queue<CompletionInfo> myCompletionInfos = new LinkedList<CompletionInfo>();
-  @NotNull private final Map<String, List<NavigationRegion>> myNavigationData = new THashMap<String, List<NavigationRegion>>();
+  @NotNull private final Map<String, List<PluginNavigationRegion>> myNavigationData = new THashMap<String, List<PluginNavigationRegion>>();
```

Comment:
Though probably it's not a problem, as we have to reimplement Find Usages anyway